CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## CHANGE LOG
 
+### v0.4.26
```

Comment:
This is a big update for the internal tooling of the app, so make it 0.5.0 instead.

Reply:
Also, bundle it together with the `0.4.25` nodemon update. That bump hasn't been "released" yet so it's fine to merge into the same minor bump.